packages/react-core/src/components/Truncate/Truncate.tsx

```diff
@@ -17,11 +18,15 @@ const truncateStyles = {
 
 const minWidthCharacters: number = 12;
 
-export interface TruncateProps extends React.HTMLProps<HTMLSpanElement> {
+export interface TruncateProps extends Omit<React.HTMLProps<HTMLSpanElement | HTMLAnchorElement>, 'ref'> {
```

Comment:
Could omitting `ref` be problematic if people were trying to use it in this component before? Do we need to make sure to forward a ref or something here?